pom.xml

```
@@ -24,8 +24,8 @@
     </licenses>
     
     <properties>
-        <jenkins.version>2.222</jenkins.version>
-        <revision>1.30</revision>
+        <jenkins.version>2.332.4</jenkins.version>
+        <revision>2.222</revision>
```

This property is the version number of the next release. It should be left as `1.38`.
```suggestion
        <revision>1.38</revision>
```